.env.example

```diff
@@ -115,6 +115,10 @@ SENDGRID_API_KEY=
 SENDGRID_EMAIL=
 NEXT_PUBLIC_SENDGRID_SENDER_NAME=
 
+# Formbricks Experience Management Integration
+# [apiHost]+[environmentId]+[surveyId]
+NEXT_PUBLIC_FORMBRICKS=
```

Comment:
I'm not sure if we should mix all of them in one environment variable. If we do we need more information about the part or rename `surveyId` to `feedbackSurveyId` to make this clear

Reply:
Done to:
```
# [apiHost]+[environmentId]
FORMBRICKS=
# Formbricks Feedback Survey ID
FORMBRICKS_FEEDBACK_SURVEY_ID=

```